### DISCUSSION TITLE
Interpolation speed of FluidValue

### DISCUSSION THREAD

**Discussion by DennisSSDev** [pmndrs/react-spring#1227] [Support]
Hi! I've been using the use gesture lib with animated div to drag it around the screen. A thing that I've noticed is the cursor is usually much faster than the dragged item and in some cases, the item dragged takes some time to catch up to the cursor. Is there any way to increase the interpolation of FluidValue numbers to help the dragged item interpolate fast towards the mouse cursor???

**Comment by aleclarson** [comment]
The `config.tension` prop sounds like what you want. See the docs: https://www.react-spring.io/docs/hooks/api

**Reply by benwest** [reply]
Or even `immediate: true`
